docs/index.md

```diff
@@ -1,54 +1,75 @@
+</select>
+<div id="aws-demo" style="display: none;">
+  {% capture aws_annotation %}
+  The full EC2 instance <code>find</code> query is <code>find . -k '*instance' -m '.state.name' running -m '.tags[?].key' owner</code>. This returns all running EC2 instances with the 'owner' tag.
+  {% endcapture %}
+
+  {% include screencast.html name="intro/aws" poster="0:17" annotation=aws_annotation %}
+</div>
+<div id="gcp-demo" style="display: none;">
+  {% capture gcp_annotation %}
+  The full compute instance <code>find</code> query is <code>find . -k '*instance' -m '.status' RUNNING -m '.labels.owner' -exists</code>. This returns all running compute instances with the 'owner' label.
+  {% endcapture %}
+
+  {% include screencast.html name="intro/gcp" poster="0:18" annotation=gcp_annotation %}
+</div>
+<div id="kubernetes-demo" style="display: none;">
+  {% capture kubernetes_annotation %}
+  The full pods <code>find</code> query is <code>find . -k '*pod' -m '.status.phase' Running -m '.metadata.labels.pod-template-hash' -exists</code>. This returns all running pods with the 'pod-template-hash' label.
+  {% endcapture %}
+
+  {% include screencast.html name="intro/kubernetes" poster="0:18" annotation=kubernetes_annotation %}
+</div>
+<div id="docker-demo" style="display: none;">
+  {% capture docker_annotation %}
+  The full container <code>find</code> query is <code>find . -k '*container' -m '.state' running -m '.labels.com\.docker\.compose\.version' -exists</code>. This returns all running containers with the 'com.docker.compose.version' label.
+  {% endcapture %}
+
+  {% include screencast.html name="intro/docker" poster="0:18" annotation=docker_annotation %}
+</div>
+<div id="external-plugin-demo" style="display: none;">
+  {% capture external_plugin_annotation %}
+  The Spotify plugin shows off Wash's greatest power: its ability to talk to <i>anything</i> via the external plugin interface. And when we say anything, we really do mean anything. We mean other cloud native vendors like OpenStack or Azure. We mean personal IoT devices like network devices, smart lightbulbs, or bluetooth-enabled espresso scales. We mean IT infrastructure like Puppet nodes or Bolt inventory files. And we also mean some truly bizarre APIs like Goodreads or Fandango. Thus if you've got some other things you'd like to <code>cd</code> and <code>ls</code> through, filter with <code>find</code>, read with <code>cat</code>, or <a href="{{ '/docs#actions' | relative_url }}">more</a>, then give Wash a try. We already have some <a href="{{ '/docs/external-plugins#example-plugins' | relative_url }}">community-built external plugins</a> that you can use. If those aren't enough, then you're more than welcome to write your own external plugin in <i>any</i> language you like (think Bash, Ruby, Python, Go). The sky is the limit.
+  {% endcapture %}
+
+  {% include screencast.html name="intro/external-plugins" poster="0:15" annotation=external_plugin_annotation %}
```

Comment:
We could also demo some other external plugins here in the future (thanks Geoff)